clang-tools-extra/clangd/ProjectModules.h

```diff
@@ -9,12 +9,17 @@
 #ifndef LLVM_CLANG_TOOLS_EXTRA_CLANGD_PROJECTMODULES_H
 #define LLVM_CLANG_TOOLS_EXTRA_CLANGD_PROJECTMODULES_H
 
+#include "support/Function.h"
 #include "support/Path.h"
 #include "support/ThreadsafeFS.h"
 
 #include <memory>
 
 namespace clang {
+namespace tooling {
+class CompileCommand;
```

Comment:
ah sorry I missed the dep on `tooling::CompileCommand`. feel free to re-add `#include "clang/Tooling/CompilationDatabase.h"` instead of the forward-decl.